internal/commands/push/command.go

```diff
@@ -546,9 +548,6 @@ func deployDraftAndWait(ui terminal.UI, realmClient realm.Client, remote appRemo
 
 			deployment, err = realmClient.Deployment(remote.GroupID, remote.AppID, deployment.ID)
 			if err != nil {
-				if e := realmClient.DiscardDraft(remote.GroupID, remote.AppID, draftID); e != nil {
```

Comment:
why isn't this needed any longer?

Reply:
🪄 ... the `err` returned from `deployDraftAndWait` brings us to the newly-created, your-not-so-favorite, `discardDraft` helper function

Reply:
helper's gone, discarding draft happens in line, but this still remains true: the `err` (from the line above here) is returned from the anonymous `waitForDeployment` func defined within `deployDraftAndWait`, which is then promptly returned and will trigger the `Handler` to discard the draft at that point (which allows us to account for completed-but-failed deployments)

Reply:
:shipit: